src/src/beneficiaries/beneficiaries.controller.ts

```diff
@@ -71,9 +71,45 @@ export class BeneficiariesController {
 		}
 	}
 
+	@Post('deactivate-duplicate-beneficiaries')
+	@UseGuards(new AuthGuard())
```

Comment:
`admin/list/deactivate-duplicates`

rename as this

Reply:
Done